src/createCli.js

```diff
@@ -33,26 +34,41 @@ const createCli = (argv) => {
     .command({
       command: 'new',
       desc: 'Create new Graasp app.',
-      builder: (_) => _.option('s', {
-        alias: 'starter',
-        type: 'string',
-        default: DEFAULT_STARTER,
-        describe: `Set starter. Defaults to ${DEFAULT_STARTER}`,
-      }).option('f', {
-        alias: 'framework',
-        type: 'string',
-        describe: 'Set development framework (e.g. React, Angular)',
-      }).option('t', {
-        alias: 'type',
-        choices: ['app', 'lab'],
-        describe: 'Type of application (app or lab)',
-      }).option('p', {
-        alias: 'path',
-        type: 'string',
-        describe: 'Path where project directory will be set up.',
-      }),
+      builder: (_) =>
+        _.option('s', {
+          alias: 'starter',
+          type: 'string',
+          default: DEFAULT_STARTER,
+          describe: `Set starter. Defaults to ${DEFAULT_STARTER}`,
+        })
+          .option('f', {
+            alias: 'framework',
+            type: 'string',
+            describe: 'Set development framework (e.g. React, Angular)',
+          })
+          .option('t', {
+            alias: 'type',
+            choices: ['app', 'lab'],
+            describe: 'Type of application (app or lab)',
+          })
+          .option('p', {
+            alias: 'path',
+            type: 'string',
+            describe: 'Path where project directory will be set up.',
+          }),
       handler: promisify(prompt),
     })
+    .command({
+      command: 'deploy',
+      desc: 'Deploy the Graasp app',
+      builder: (_) =>
+        _.option('p', {
+          alias: 'path',
+          type: 'string',
+          describe: 'Path to the Graasp app that shall be deployed',
+        }),
+      handler: promisify(deploy),
```

Comment:
Do you need to `promisify`? The function is already a promise. Just double-checking.

Reply:
@juancarlosfarah I'm actually not sure. This is inspired by the implementation of the `new` command. But I tested now and the `promisify()` could be omitted in both cases. Please let me know if I'm missing a point here, otherwise I'll remove it in the next PR.

Reply:
Maybe originally the function was not a promise, thus `promisify` was needed. Looks like it's not needed anymore.